sw/device/silicon_creator/mask_rom/meson.build

```diff
@@ -33,7 +33,10 @@ mask_rom_lib = declare_dependency(
     ],
     link_with: static_library(
       'mask_rom_lib',
-      sources: ['mask_rom.c'],
+      sources: [
+        rom_exts_manifest_offsets_header,
```

Comment:
For me, just this line was sufficient to resolve the build failure.

Reply:
Good to know. I must have just made a typo then.

Reply:
So what are your thoughts on merging this as is?  Look good to you?

Reply:
Or I guess as a question to @moidx, are you intending to merge this test PR as is?

Reply:
SG. Updated the commit to only include this change.